test/integration/OrderBook.spec.ts

```diff
@@ -30,23 +30,23 @@ const initValues = async (db: DB) => {
   ]);
 };
 
+const sandbox = sinon.createSandbox();
```

Comment:
What do you think about moving `nomatching OrderBook` describe block within `Orderbook` instead of moving the scope of sandbox & pool to outer scope?

Reply:
I thought of this same thing, but I kind of liked having the nomatching tests separate. Although in thinking about this a bit more, maybe it messes up how we call `sandbox.restore()` in the `after` block. I can keep nomatching and matching separate, but have each have a separate `sandbox` within their scope. Does that sound good to you?

Reply:
That works.

Minor nit: I also noticed `Orderbook` describe block has 2 after blocks which probably means one of them isn't being executed.

Reply:
Done.